rsearch/config.ini

```diff
@@ -0,0 +1,18 @@
+; comment
+[api]
+url=http://192.168.0.10:8080
```

Comment:
Kube api url does change, namespace endpoint might not change but we want to be able to define api url separately as it's going to be used in few places.